WordPress/Classes/Stores/PluginStore.swift

```diff
@@ -334,6 +403,37 @@ private extension PluginStore {
         })
     }
 
+    func installPlugin(slug: String, site: JetpackSiteRef) {
+        guard let remote = remote(site: site),
+            !state.updatesInProgress[site, default: Set()].contains(slug),
+            getPlugin(slug: slug, site: site) == nil else {
+                return
+        }
+
+        transaction { state in
+            state.updatesInProgress[site, default: Set()].insert(slug)
+        }
+
+        remote.install(
+            pluginSlug: slug,
+            siteID: site.siteID,
+            success: { [weak self] plugin in
+                self?.transaction { state in
+                    state.plugins[site]?.plugins.append(plugin)
```

Comment:
Does this ensure there are no duplicates? I'm thinking about something triggering a plugin refresh right before the install response is received. Just like we have `modifyPlugin`, I'm thinking of an `insertOrUpdate` method (although I don't love that specific name)